### ISSUE TITLE
Allow for custom file extensions

### ISSUE THREAD

**Issue by bemeurer** [oliverkeeble/atom-language-assembly#6]
I have a lot of assembly code with weird extensions due to some bad design decisions made before my time. Mostly code with the `.as` extension as opposed to `.asm`, which means the plugin won't highlight them. Could you add the `.as` extension or add a field to support custom ones?

**Comment by oliverkeeble** [comment]
I've just added support for `.as` files for you as a temporary fix and published v0.7.2, but I'll leave this issue open for now as I would like to add support for custom file extensions in the future.
Thanks.

**Comment by oliverkeeble** [comment]
I'm closing this now as this has been implemented in Atom. You can add custom extensions by editing your config file. For example:

```coffeescript
"*":
  core:
    customFileTypes:
      "source.assembly.asm.x86.intel": [
        "as"
      ]
```
